crates/db_views/src/site_view.rs

```diff
@@ -9,22 +9,21 @@ use lemmy_db_schema::{
 impl SiteView {
   pub async fn read_local(pool: &mut DbPool<'_>) -> Result<Self, Error> {
     let conn = &mut get_conn(pool).await?;
-    let mut res = site::table
-      .inner_join(local_site::table)
-      .inner_join(
-        local_site_rate_limit::table.on(local_site::id.eq(local_site_rate_limit::local_site_id)),
-      )
-      .inner_join(site_aggregates::table)
-      .select((
-        site::all_columns,
-        local_site::all_columns,
-        local_site_rate_limit::all_columns,
-        site_aggregates::all_columns,
-      ))
-      .first::<SiteView>(conn)
-      .await?;
-
-    res.site.private_key = None;
-    Ok(res)
```

Comment:
The output of this function is used for signed fetch, and it would crash due to missing private key. Actually there should be no need to set it as None, we can simply use serde(skip).